src/wordvec.c

```diff
@@ -0,0 +1,50 @@
+#define _GNU_SOURCE
+#include "wordvec.h"
+
+struct wordvec *wordvec_new(void)
+{
+	struct wordvec *wv = malloc(sizeof(*wv));
+	if (!wv)
+		return NULL;
+
+	wv->len = 0;
+	wv->cap = 8;
```

Comment:
```suggestion
	wv->cap = WORDVEC_DEFAULT_CAP;
```
Maybe define a macro in the C file for easier changes?